### ISSUE TITLE
Allow `from` to be array of paths / glob, respect order

### ISSUE THREAD

**Issue by emkayy** [webpack-contrib/copy-webpack-plugin#631]
### Feature Proposal

Please allow the **from** property to contain an array of paths / glob patterns
because I need to include files from several different locations and copy them all into one folder.
This is quite ugly and verbose doing it with glob.

The assets array in transformAll should then be returned **in the order of the from array** so you can rely on javascript execution order.

### Feature Use Case
Copying and or concatenating and transforming (javascript) files from multiple locations into one file / folder.

### Please paste the results of `npx webpack-cli info` here, and mention other relevant information
not relevant

**Comment by alexander-akait** [comment]
Respecting order is very bad for perf, work is going on on different cores, please sort them inside `transformAll` as you want, sorry, we don't want to implement this

**Comment by alexander-akait** [comment]
Because this logic don't know nothing about of input order

**Comment by alexander-akait** [comment]
> from as an array

Yes, we can do it, make sense, but why do not use blog base above and using `filter` function, better DX?
